src/cli/simple_client_manager.py

```diff
@@ -9,14 +9,17 @@ def __init__(self, client_path, node_addr, verbose=None) -> None:
         self.client_path = client_path
         self.cmd_manager = CommandManager(verbose)
         self.node_hostname = "127.0.0.1"
-        self.node_port  =  8732
```

Comment:
I am not happy with leaving "magic numbers" anywhere in the code. I suggest to put the standard port into a global constant and only use the constant in the code.